psc_verification.yml

```diff
@@ -18,6 +18,7 @@ validation:
   'dob-mismatch': "Date of birth does not match"
   'forenames-mismatch': "Forename(s) do not match: a name mismatch reason must be provided"
   'surname-mismatch': "Surname does not match: a name mismatch reason must be provided"
+  'no-name-mismatch-reason' : "The name on the public register is different to the name this PSC used for identity verification: a name mismatch reason must be provided"
```

Comment:
how trivial is this.. but there's an extra space before the colon.  Approving now anyway.

Reply:
Removed space as a precaution.